roles/kubernetes/control-plane/tasks/psp-install.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Check AppArmor status
+  command: which apparmor_parser
```

Comment:
https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/2405847732
```
roles/kubernetes/control-plane/tasks/psp-install.yml:2: no-changed-when Commands should not change things if nothing needs doing
```
Need to add changed_when into this task to pass ansible-lint job like
```
changed_when: apparmor_status.rc != 0
```